pkg/io/OWNERS

```diff
@@ -0,0 +1,4 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
```

Comment:
Please add the other relevant approvers. This will make you the only person who ever gets assigned io changes.

Reply:
Its definitely only used by prow:
```
$ ack -Q pkg/io -l|egrep -v '^prow'
BUILD.bazel
pkg/io/providers/providers_test.go
pkg/io/providers/BUILD.bazel
pkg/io/opener.go
pkg/io/BUILD.bazel
```

Do you mind if I instead just move the package under `prow/`?

Reply:
SGTM